src/core/trulens/core/schema/base.py

```diff
@@ -36,9 +36,15 @@ class Cost(SerialModel, pydantic.BaseModel):
     n_completion_tokens: int = 0
     """Number of completion tokens generated."""
 
+    n_guardrails_tokens: int = 0
```

Comment:
We are filling this, but not using anywhere?

Reply:
it's still worthwhile to report guardrail token usage for Cortex user.